iroh-bytes/src/protocol.rs

```diff
@@ -19,19 +18,8 @@ use crate::util::Hash;
 /// Maximum message size is limited to 100MiB for now.
 pub(crate) const MAX_MESSAGE_SIZE: usize = 1024 * 1024 * 100;
 
-/// Protocol version
-pub const VERSION: u64 = 2;
-
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq, Clone, MaxSize)]
-pub(crate) struct Handshake {
-    pub version: u64,
-}
-
-impl Handshake {
-    pub fn new() -> Self {
-        Self { version: VERSION }
-    }
-}
+/// The ALPN used with quic for the iroh bytes protocol.
+pub const ALPN: [u8; 13] = *b"/iroh-bytes/2";
```

Comment:
you want to lose the `n0/` prefix?

Reply:
yeah, seems unnecessary to me,

Reply:
but why leave the leading `/`?

Reply:
personal preference, not a strong one though